lib/Parse/Parser.cpp

```diff
@@ -1348,8 +1348,8 @@ DeclNameRef swift::formDeclNameRef(ASTContext &ctx,
   // We cannot import when the base name is not an identifier.
   if (baseName.empty())
     return DeclNameRef();
-  if (!Lexer::isIdentifier(baseName) && !Lexer::isOperator(baseName))
-    return DeclNameRef();
+//  if (!Lexer::isIdentifier(baseName) && !Lexer::isOperator(baseName))
+//    return DeclNameRef();
```

Comment:
I will need to find a better condition here. I will probably just make this an assert and put it elsewhere.

Reply:
I haven't forgotten about this, just want to see how we're doing on tests.

Reply:
I actually don't know how helpful this behavior is. I am going to remove this from the patch and if the code owners feel that we need further validation of names then let's add this as an assert after the fact.